packages/crud/src/vaadin-crud.js

```diff
@@ -945,6 +937,32 @@ class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement)))
     }
   }
 
+  /**
+   * @param {HTMLElement | undefined} form
+   * @param {string} theme
+   * @param {string | string[] | undefined} include
+   * @param {string | RegExp} exclude
+   * @private
+   */
+  // eslint-disable-next-line max-params
+  __gridPropsChanged(grid, theme, include, exclude, noFilter, noHead, noSort) {
+    if (grid) {
+      if (grid === this._gridController.defaultNode) {
+        grid.include = include;
+        grid.exclude = exclude;
+        grid.noFilter = noFilter;
+        grid.noHead = noHead;
+        grid.noSort = noSort;
+      }
+
+      if (theme) {
+        grid.setAttribute('theme', theme);
+      } else {
+        grid.removeAttribute('theme');
+      }
```

Comment:
It looks like the theme was not propagated to the slotted grid before? I think it makes sense to propagate it, but want to make sure I'm getting this right.

Reply:
Yes, sounds like it wasn't. I could create a separate PR for `23.3` branch to fix that.

Reply:
UPD: actually, we have ITs in Flow counterpart to ensure theme is not propagated. Created #4999 to fix.